insertBaseData-psql.sql

```diff
@@ -1147,6 +1150,52 @@ INSERT INTO "noteRelatedObjects" ("noteUuid", "relatedObjectType", "relatedObjec
+	VALUES (uuid_generate_v4(), :authorUuid, 'myAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO attachments (uuid, "authorUuid", "fileName", "mimeType", content, "contentLength", "description", "classification", "createdAt", "updatedAt")
+	VALUES (uuid_generate_v4(), :authorUuid, 'myAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO attachments (uuid, "authorUuid", "fileName", "mimeType", content, "contentLength", "description", "classification", "createdAt", "updatedAt")
+	VALUES (uuid_generate_v4(), :authorUuid, 'myAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO attachments (uuid, "authorUuid", "fileName", "mimeType", content, "contentLength", "description", "classification", "createdAt", "updatedAt")
+	VALUES (uuid_generate_v4(), :authorUuid, 'myAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO attachments (uuid, "authorUuid", "fileName", "mimeType", content, "contentLength", "description", "classification", "createdAt", "updatedAt")
+	VALUES (uuid_generate_v4(), :authorUuid, 'myAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO attachments (uuid, "authorUuid", "fileName", "mimeType", content, "contentLength", "description", "classification", "createdAt", "updatedAt")
+	VALUES (uuid_generate_v4(), :authorUuid, 'myAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO attachments (uuid, "authorUuid", "fileName", "mimeType", content, "contentLength", "description", "classification", "createdAt", "updatedAt")
+	VALUES (uuid_generate_v4(), :authorUuid, 'myAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+
+-- Add attachments for reports
+SELECT ('''' || uuid || '''') AS "authorUuid" FROM people WHERE name = 'DMIN, Arthur' \gset
+SELECT ('''' || uuid_generate_v4() || '''') AS "attachmentUuid" \gset
+INSERT INTO attachments (uuid, "authorUuid", "fileName", "mimeType", content, "contentLength", "description", "classification", "createdAt", "updatedAt")
+	VALUES (:attachmentUuid, :authorUuid, 'myNewAttachment', 'image/jpeg', null, 1000, 'We can add attachment for report', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO "attachmentRelatedObjects" ("attachmentUuid", "relatedObjectType", "relatedObjectUuid")
+  SELECT :attachmentUuid, 'reports', r.uuid
+  FROM reports r
+  WHERE r.intent = 'Test report with rich text';
+SELECT ('''' || uuid_generate_v4() || '''') AS "attachmentUuid" \gset
```

Comment:
Why do you generate a new `attachmentUuid` if you're going to use a static value of `'f076406f-1a9b-4fc9-8ab2-cd2a138ec26d'` anyway?